ocs_ci/utility/utils.py

```diff
@@ -1071,6 +1072,11 @@ def get_testrun_name():
             {key: '-' for key in ''' \\/.:*"<>|~!@#$?%^&'*(){}+`,=\t'''}
         )
     )
+    # form complete testrun name that includes deployment platform and type
+    testrun_name = (
+        f"{testrun_name} - {config.ENV_DATA.get('platform').upper()}-"
+        f"{config.ENV_DATA.get('deployment_type').upper()}"
+    )
```

Comment:
Please move the lines 1075-1079 before line 1069, so replacing invalid characters will apply to the whole  `testrun_name` (the replaced characters breaks Polarion import process).